packages/ui/src/apis/utils.ts

```diff
@@ -15,32 +15,32 @@ import {
 
 export type PasscodeChannel = 'sms' | 'email';
 
-export const getSendPasscodeApi = (type: UserFlow, channel: PasscodeChannel) => {
-  if (type === 'sign-in' && channel === 'email') {
+export const getSendPasscodeApi = (type: UserFlow, method: PasscodeChannel) => {
+  if (type === 'sign-in' && method === 'email') {
     return sendSignInEmailPasscode;
   }
 
-  if (type === 'sign-in' && channel === 'sms') {
+  if (type === 'sign-in' && method === 'sms') {
     return sendSignInSMSPasscode;
   }
 
-  if (type === 'register' && channel === 'email') {
+  if (type === 'register' && method === 'email') {
     return sendRegisterEmailPasscode;
   }
 
   return sendRegisterSMSPasscode;
 };
```

Comment:
seems that always return `sendSignInEmailPasscode` when `method` is `email` and return `sendRegisterSMSPasscode` when `method` is `sms`.

could we simplify it? (maybe do it in another PR)

Reply:
sorry not following, what do you mean by `always return sendSignInEmailPasscode when method is email` ? There is another API call sendRegisterXXXXPasscode